lerna.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
@ntwb, we really need to add Eslint to make sure we don't need to deal with all that code formatting issues like tabs vs spaces.

Reply:
ESLint added at any stage as in after this PR would easily address these. I'll make a separate PR for that today :)

Reply:
Yes, that's why I pinged Stephen.

Reply:
I have a working copy inside create-guten-block if you'd like to take a look at it?

Reply:
👌

Reply:
Yup, I'll take care of ESLint as stated in the other PR in the coming days